tools/tune/dcblock/example_dcblock.m

```diff
@@ -0,0 +1,23 @@
+function example_dcblock();
+
+% Set the parameters here
+tplg_fn = "../../topology/m4/dcblock_coef_default.m4" % Control Bytes File
+% Use those files with sof-ctl to update the component's configuration
+blob_fn = "../../ctl/dcblock_coef.blob" % Blob binary file
+alsa_fn = "../../ctl/dcblock_coef.txt" % ALSA CSV format file
+
+endian = "little";
+R_coeffs = [0.98, 0.98, 0.98, 0.98, 0.98, 0.98, 0.98, 0.98];
```

Comment:
Could you work out the transfer function for 1st order filter coefficients "b" and "a" for Octave function freqz() usage so it could be possible to plot the frequency response (especially to know the -3 dB cutoff frequency) and step response computed with  for the chosen R sample rate.

Something like this (I used butter())

```
fs=48e3;
f=linspace(1, fs/2, 500);
[b,a]=butter(1, 2*40/fs, 'high');
semilogx(f, 20*log10(freqz(b, a, f, fs)))
grid on
figure
t=((1:0.5*fs)-1)/fs;
plot(t, filter(b, a, ones(length(t), 1)))
grid on

```